packages/subcommands/src/lib/SubcommandMappings.ts

```diff
@@ -0,0 +1,123 @@
+import type { Args, Awaitable, ChatInputCommand, MessageCommand } from '@sapphire/framework';
+import type { Message } from 'discord.js';
+
+export type SubcommandMappingsArray = (
+	| ChatInputSubcommandGroupMappings
+	| ChatInputSubcommandMappings
+	| MessageSubcommandMappings
+	| MessageSubcommandGroupMappings
+)[];
+export type ChatInputSubcommandToProperty = (interaction: ChatInputCommand.Interaction, context: ChatInputCommand.RunContext) => Awaitable<unknown>;
+export type MessageSubcommandToProperty = (message: Message, args: Args, context: MessageCommand.RunContext) => Awaitable<unknown>;
+export type SubcommandType = 'method' | 'command';
+export class ChatInputSubcommandGroupMappings {
```

Comment:
```suggestion
)[];

export type ChatInputSubcommandToProperty = (interaction: ChatInputCommand.Interaction, context: ChatInputCommand.RunContext) => Awaitable<unknown>;
export type MessageSubcommandToProperty = (message: Message, args: Args, context: MessageCommand.RunContext) => Awaitable<unknown>;
export type SubcommandType = 'method' | 'command';

export class ChatInputSubcommandGroupMappings {
```

Reply:
the diff wants us to add a newline on line 13->14

because github

Reply:
yeah because I added it?

Reply:
was commenting because sometimes these diffs are stupidly hard to read